app/matchers/LanguageToolMatcher.scala

```diff
@@ -75,8 +73,11 @@ class LanguageToolFactory(
       case Success(_) => None
       case Failure(e) => {
         logger.error(e.getMessage(), e)
-        Some(e.getMessage())
+        Some(e)
       }
+    } match {
+      case Nil => Right(())
+      case errors => Left(errors)
```

Comment:
This isn't very elegant but can't immediately think of ways to improve it. I suspect using `Validation` might help - I wonder if ZIO has such a thing or if we'd need to lean on cats.

Reply:
Yeah, it's a bit frustrating. Interested to see if a common pattern for effects with failure modes helps resolve